tree/tree/src/TTree.cxx

```diff
@@ -1013,9 +1024,6 @@ TTree::~TTree()
    fClusterRangeEnd = 0;
    delete [] fClusterSize;
    fClusterSize = 0;
-   // Must be done after the destruction of friends.
```

Comment:
Why does

> // Must be done after the destruction of friends.

not hold anymore?

Reply:
See commit message: "Delaying is no longer necessary since the introduction of TFriendElement::fOwnFile in commit 115203c"  ... ie. since 20 years ago !!!